# Generated by Django 2.1.7 on 2019-03-04 12:30

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0003_auto_20190221_2228'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='account',
            name='action',
        ),
        migrations.RemoveField(
            model_name='account',
            name='emotion',
        ),
        migrations.AlterField(
            model_name='account',
            name='username',
            field=models.CharField(error_messages={'unique': 'そのアカウントIDを持ったアカウントはすでに存在しています'}, help_text='Required. 31 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=31, unique=True, validators=[django.core.validators.RegexValidator('^[0-9a-zA-Z]*$', '英数字のみしか利用できません.')], verbose_name='アカウントID'),
        ),
    ]
